force-app/main/default/lwc/routingService/routingService.js

```diff
@@ -56,8 +56,8 @@ export function updateRawLinkInfo(element, { url, target }) {
         // eslint-disable-next-line no-console
         console.error('url must be specified');
     }
-    if (target === '_blank') {
-        return new Promise(resolve => {
+    if (target && target !== '_self') {
+        return new Promise((resolve) => {
```

Comment:
I'm seeing a discrepancy in this file compared to the original file - https://github.com/salesforce/lightning-components/blob/master/ui-lightning-components/src/main/modules/lightning/routingService/routingService.js

@sgesin Can you confirm if the changes here are expected or if we need to update this file?

Reply:
@vbhargavmbis It should match with the 228/patch version:
https://github.com/salesforce/lightning-components/blob/228/patch/ui-lightning-components/src/main/modules/lightning/routingService/routingService.js

It seems like the only diffs between those two is formatting and comments getting stripped.

Reply:
Ah ok, I was looking at 230. Cool.